### PR TITLE
Update to 61.2.0

### PR DESCRIPTION
**Pull request by tobijk**
Release branch:
https://github.com/pypa/setuptools/tree/v61.2.0

Changelog:
https://github.com/pypa/setuptools/blob/v61.2.0/CHANGES.rst

Modifications:
* Pointed at new sources
* Rebased/updated patches
* Removed patches that weren't applicable anymore

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,35 +7,29 @@ package:
...
-    - patches/0003-Win32-distutils-Add-support-to-cygwinccompiler-for-V.patch
-    - patches/0010-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch  # [py<=39 and python_impl=="cpython"]
     - patches/0006-Win32-Fixes-for-Windows-GCC-interop-needed-by-RPy2-a.patch
-    - patches/0034-distutils-fix-ldshared.patch
     # distutils patches from pypy3.6-feedstock
     - patches/0035-pypy-distutils-scheme.patch   # [win or py>37]
 
 build:
-  number: 1
+  number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:23]: We should skip `py<37` https://github.com/pypa/setuptools/blob/0ec53b228800300992ba1c53c2f089a435d4970c/setup.cfg#L27

**Review by ScottBailey — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (ScottBailey) [reply recipe/meta.yaml:23]: We've removed py36 and below from our build process, so pedantically: yes we should `skip: true  # [py<37]`; pragmatically we already do. Wouldn't it be just as good to remove these?

Comment (skupr-anaconda) [reply recipe/meta.yaml:23]: I think it's for clarification if others want to know constraints about supported python versions

**Review by varlackc — COMMENTED** [review]
Other than the requested changes everything else looks good. Once the changes are made we should be good to approve.

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:23]: https://github.com/pypa/setuptools/issues/2930

Comment (tobijk) [reply recipe/meta.yaml:23]: Thanks for bringing this up and starting the discussion, @skupr-anaconda.

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Comment by skupr-anaconda** [comment]
Hmm, E`rror: HTTP 503 BACKEND IS UNHEALTHY for url <https://pypi.io/packages/source/s/setuptools/setuptools-61.2.0.tar.gz>`

**Comment by skupr-anaconda** [comment]
Hmm, `Error: HTTP 503 BACKEND IS UNHEALTHY for url <https://pypi.io/packages/source/s/setuptools/setuptools-61.2.0.tar.gz>`

**Pull request merged**